.travis.yml

```diff
@@ -1,4 +1,6 @@
 language: node_js
+before_install:
+- npm install -g npm@latest
```

Comment:
I can't repro or find any reference to npm versioning/iojs, and the tests seem to be passing. If anyone knows better I'm all ears